packages/perseus/src/util/parse-perseus-json/perseus-parsers/numeric-input-widget.ts

```diff
@@ -57,7 +57,7 @@ export const parseNumericInputWidget: Parser<NumericInputWidget> = parseWidget(
         ),
         labelText: optional(string),
         size: string,
-        coefficient: boolean,
+        coefficient: defaulted(boolean, () => false),
```

Comment:
Going through these regression tests has started me thinking. The widget React components often provide default props which fill in these missing values (eg. `coefficient` is defaulted to `false` [here](https://github.com/Khan/perseus/blob/d93e3ecdeb6bd714a35dcd9f886299fa80ba71ec/packages/perseus/src/widgets/numeric-input/numeric-input.tsx#L107)).

Sorry, I have post-vacation brain, but have we discussed this before?

Reply:
My thoughts on where to put defaults are described in the [parse-perseus-json/README.md](https://github.com/Khan/perseus/blob/265a9310486e5c1524af9b502619db9de2f7c01d/packages/perseus/src/util/parse-perseus-json/README.md#default-values). I think we want them in both the parser and the component's default props, because those defaults serve different purposes.